packages/server/src/types.ts

```diff
@@ -147,7 +148,7 @@ export interface ApolloServerOptionsWithTypeDefs<TContext extends BaseContext>
   // makeExecutableSchema. (If you don't want to use makeExecutableSchema, pass
   // `schema` instead.)
   typeDefs: IExecutableSchemaDefinition<TContext>['typeDefs'];
-  resolvers?: IExecutableSchemaDefinition<TContext>['resolvers'];
+  resolvers?: IResolvers<any, TContext>;
```

Comment:
The change here is to not allow an array of resolvers objects? Is this necessary? I kinda like that for our pass-through arguments we just allow whatever the underlying function takes. Was that what caused the weird issue we saw where context wasn't being inferred correctly in resolver bodies?

Reply:
Yeah this is just a stray change from when we were investigating the context not being threaded all the way through the resolvers.

Reply:
[f6b44fd](https://github.com/apollographql/apollo-server/pull/6420/commits/f6b44fd4e533e3abad21f0bd4029b816888a9bf3)